docs/source/private-data-arch.rst

```diff
@@ -183,6 +210,28 @@ along with the data in the chaincode APIs, for example
 
 A single chaincode can reference multiple collections.
 
+Referencing implicit collections from chaincode
+-----------------------------------------------
+
+Starting in v2.0, an implicit private data collection exists for each organization
+in a channel, so that you don't have to define collections if you'd like to
+utilize per-organization collections. Each org-specific implicit collection
```

Comment:
Would this be `an implicit private data collection is automatically created for each organization
in a channel when the organization joins the channel.`?

What happens if an organization leaves the channel, does the implicit collection go away/is removed?
I see this text below: `Note that collections cannot be deleted, as there may be prior private data hashes on the channel’s blockchain that cannot be removed.`

Reply:
I updated the text a bit... implicit private data collections don't really exist until they are used.
And yes, just like regular private data collections, they will remain even after an org is removed from a channel since there are corresponding hashes on the blockchain.

Reply:
No, the implicit collection is not deleted when an organization leave the channel. The other organizations would still have hashes of the private data in this collection.